test/spec/converters/json_to_jdl_entity_converter_test.js

```diff
@@ -251,6 +251,24 @@ describe('JSONToJDLEntityConverter', () => {
       });
     });
   });
+
+  describe('::convertEntitiesToJDL without relationship', () => {
```

Comment:
It should be a `context` case inside the `describe('::convertBlaBlaBla` up this file.
Like this:
```javascript
context('without relationships', () => {
  // your declarations & setup
  it('....', () => {

  });
});
```

I don't quite understand what you're testing here: that it doesn't fail or the fact that there's a table name?

Reply:
Yes, exactly.
Without the condition in `lib/converters/json_to_jdl_entity_converter.js`, this test will fail, as it can't convert to JDL